src/main/resources/assets/sodium/shaders/blocks/block_layer_opaque.vsh

```diff
@@ -20,9 +20,19 @@ uniform vec3 u_RegionOffset;
 
 uniform sampler2D u_LightTex; // The light map texture
 
+#if defined(VERTEX_FORMAT_COMPACT)
+// expect 0-15 as inputs, Sodium 0.5 logic
 vec3 _sample_lightmap(uvec2 coord) {
     return texelFetch(u_LightTex, ivec2(coord), 0).rgb;
 }
+#elif defined(VERTEX_FORMAT_FULL)
```

Comment:
I think this could use `#ifdef` but I'm not sure, doesn't really make a difference.